lib/browser/rpc-server.js

```diff
@@ -257,148 +257,118 @@ const callFunction = function (event, contextId, func, caller, args) {
   }
 }
 
-ipcMain.on('ELECTRON_BROWSER_REQUIRE', function (event, contextId, module) {
-  try {
-    event.returnValue = valueToMeta(event.sender, contextId, process.mainModule.require(module))
-  } catch (error) {
-    event.returnValue = exceptionToMeta(event.sender, contextId, error)
-  }
-})
+const handleRemoteCommand = function (channel, handler) {
+  ipcMain.on(channel, (event, contextId, ...args) => {
+    let returnValue
+    try {
+      returnValue = handler(event, contextId, ...args)
+    } catch (error) {
+      returnValue = exceptionToMeta(event.sender, contextId, error)
+    }
+    if (returnValue !== undefined) {
```

Comment:
This is a change in behavior, not sure if it matters but before even if it was `undefined` we set the returnValue.  Not sure if it affects anything but was this a deliberate change?

Reply:
@MarshallOfSound this should actually preserve the behavior as in some cases the IPC event handlers don't set the returnValue

Reply:
> even if it was `undefined` we set the `returnValue`
@MarshallOfSound 
`valueToMeta` and `exceptionToMeta` never return `undefined`, and I see not explicit assignments of `undefined` to `event.returnValue`.
So I don't see any changes in behaviour here.